diff --git a/node_modules/react-native-audio-api/android/build.gradle b/node_modules/react-native-audio-api/android/build.gradle
index 49b6f51..b70cbde 100644
--- a/node_modules/react-native-audio-api/android/build.gradle
+++ b/node_modules/react-native-audio-api/android/build.gradle
@@ -14,7 +14,8 @@ buildscript {
 
 def reactNativeArchitectures() {
   def value = rootProject.getProperties().get("reactNativeArchitectures")
-  return value ? value.split(",") : ["armeabi-v7a", "arm64-v8a", "x86_64", "x86"]
+  def architectures = value ? value.split(",") : ["armeabi-v7a", "arm64-v8a", "x86_64", "x86"]
+  return architectures.findAll { it != "armeabi-v7a" }
 }
 
 def isNewArchitectureEnabled() {
